pkg/gen/dotnet-templates/Provider.cs

```diff
@@ -86,5 +86,15 @@ public sealed class ProviderArgs : ResourceArgs
         /// </summary>
         [Input("suppressDeprecationWarnings")]
         public Input<bool>? SuppressDeprecationWarnings { get; set; }
+
+        /// <summary>
+        /// If present, render resource manifests to this directory. In this mode, resources will not
+        /// be created on a Kubernetes cluster, but the rendered manifests will be kept in sync with changes
+        /// to the Pulumi program. Note that some computed Outputs such as status fields will not be populated
+        /// since the resources are not created on a Kubernetes cluster. Attempting to reference these Outputs
+        /// may result in an error, or the value may be empty/undefined.
```

Comment:
Which of these is true - is it an error or will it return `undefined ` (or `null`)?

Reply:
Output values that are normally populated by the k8s cluster will be `undefined`. I'll update the docs.